assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -87,12 +107,20 @@ export default function UserInputQuestionnaire() {
 	const isSettings = single === 'settings';
 
 	const next = useCallback( () => {
-		trackEvent( viewContext, 'question_advance', steps[ activeSlugIndex ] );
+		trackEvent(
+			`${ viewContext }_kmw`,
```

Comment:
`${ viewContext }_kmw` is used so much here, is it worth just making a variable out of it and using that? 🤷🏻‍♂️